spatialos-sdk/src/worker/snapshot.rs

```diff
@@ -5,12 +5,39 @@ use crate::{
 use spatialos_sdk_sys::worker::*;
 use std::{ffi::CString, path::Path};
 
+#[derive(Debug)]
+pub enum SnapshotError {
+    Bad,
```

Comment:
Bad what? 😂

Reply:
Well, this was supposed to be a "bad state". I'll change it to `BadState`